src/applications/static-pages/covidFaqLink.js

```diff
@@ -15,7 +13,7 @@ export default function addLinkToCovidFAQ(store) {
   let linkAdded = false;
 
   store.subscribe(() => {
-    if (!linkAdded && toggleValues(store.getState()).covid19FaqChatbotLink) {
+    if (!linkAdded) {
```

Comment:
I opened a PR for this - https://github.com/department-of-veterans-affairs/vets-website/pull/12522 to remove all of the code instead of just making it return false all the time

Reply:
Ah @ncksllvn thanks!